galley/pkg/mcp/snapshot/inmemory_test.go

```diff
@@ -0,0 +1,70 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package snapshot
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+
+	"istio.io/api/config/mcp/v1alpha1"
```

Comment:
nit: can we use consistent package aliasing throughout, e.g. `mcp "istio.io/api/config/mcp/v1alpha1"`?